pkg/apis/kops/cluster.go

```diff
@@ -136,6 +136,12 @@ type ClusterSpec struct {
 	//   missing: default policy (currently OS security upgrades that do not require a reboot)
 	UpdatePolicy *string `json:"updatePolicy,omitempty"`
 
+	// Install Helm
+	// Valid values:
+	//   'true' Install the tiller manifest to enable helm https://github.com/kubernetes/helm
+	//   'false' Don't install it
+	EnableHelm *bool `json:"enableHelm,omitempty"`
```

Comment:
I think just a list of additional add-ons to load would be more versatile, i.e. something that accepted `[helm-tiller, dashboard, ...]`. You then have a semantic question of what happens if we want to add an add-on by default, and how that interacts with upgrades (i.e. upgrading a cluster from one add-on set `[A,B,C]`, but then we choose to inject a new addon `D` in a mythical default set. You can either keep the cluster as `[A,B,C]`, or go ahead and make it `[A,B,C,D]`.) Thoughts, @justinsb?